docs/how-tos/provision.md

```diff
@@ -0,0 +1,310 @@
+
+{{< tabs >}}
+{{% tab name="x86_64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+mkdir -p /opt/viam/bin/ /opt/viam/tmp/
+curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/viam-agent-stable-x86_64 -o /opt/viam/bin/viam-agent-stable-x86_64
+curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent-provisioning/viam-agent-provisioning-stable-x86_64 -o /opt/viam/bin/viam-agent-provisioning-stable-x86_64
+chmod 755 /opt/viam/tmp/viam-agent*
+```
+
+{{% /tab %}}
+{{% tab name="aarch64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+mkdir -p /opt/viam/bin/ /opt/viam/tmp/
+curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/viam-agent-stable-aarch64 -o /opt/viam/bin/viam-agent-stable-aarch64
+curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent-provisioning/viam-agent-provisioning-stable-aarch64 -o /opt/viam/bin/viam-agent-provisioning-stable-aarch64
+chmod 755 /opt/viam/tmp/viam-agent*
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< /tablestep >}}
+{{< tablestep >}}
+**2. Symlink the binaries**
+
+Symlink the agent binary to `bin/viam-agent` and the provisioning binary to `bin/agent-provisioning`:
+
+{{< tabs >}}
+{{% tab name="x86_64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+ln -s /opt/viam/bin/viam-agent-stable-x86_64 bin/viam-agent
+ln -s /opt/viam/bin/viam-agent-provisioning-stable-x86_64 bin/agent-provisioning
+```
+
+{{% /tab %}}
+{{% tab name="aarch64" %}}
+
+```sh {class="command-line" data-prompt="$"}
+ln -s /opt/viam/bin/viam-agent-stable-aarch64 bin/viam-agent
+ln -s /opt/viam/bin/viam-agent-provisioning-stable-aarch64 bin/agent-provisioning
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< alert title="Note" color="note" >}}
+Use relative symlinks, especially if working on a mounted image (that is not a booted system).
+{{< /alert >}}
+
+{{< /tablestep >}}
+{{< tablestep >}}
+**3. Create the systemd service file**
+
+Copy the systemd [service file](https://github.com/viamrobotics/agent/blob/main/subsystems/viamagent/viam-agent.service) from the agent repo to `/etc/systemd/system/viam-agent.service`.
+
+Then, symlink the service file to <FILE>/etc/systemd/system/multi-user.target.wants/viam-agent.service</FILE>
+
+```sh {class="command-line" data-prompt="$"}
+curl -fsSL https://github.com/viamrobotics/agent/raw/main/subsystems/viamagent/viam-agent.service -o /etc/systemd/system/viam-agent.service
+ln -s /etc/systemd/system/viam-agent.service /etc/systemd/system/multi-user.target.wants/viam-agent.service
+```
+
+{{< alert title="Note" color="note" >}}
+Use relative symlinks, especially if working on a mounted image (that is not a booted system).
+{{< /alert >}}
+
+{{< /tablestep >}}
+{{< tablestep >}}
+**4. Ensure NetworkManager is installed**
+
+Make sure NetworkManager is installed and enabled in `systemd`.
```

Comment:
Also that it's version 1.42 or newer for full support. (May work with slightly older versions, but not fully tested.)